### ISSUE TITLE
Make it possible to edit the spec via email

### ISSUE THREAD

**Issue by vinhowe** [vinhowe/lifesystem#17]
Waiting on #3--Perora will need to be fundamentally restructure to run behind a REST (or GraphQL) API. This is perfect for Google Cloud Run/AWS Lamda-type serverless services.

**Comment by vinhowe** [comment]
This might be fun for lifelab, but it's out of scope for this project now.
